icebram/icebram.cc

```diff
@@ -182,16 +192,12 @@ int main(int argc, char **argv)
 		xorshift64star();
 		xorshift64star();
 
-		if (!seed){
+		if (!seed) {
 			struct timeval tv;
 			gettimeofday(&tv, NULL);
 			x ^= uint64_t(tv.tv_sec) << 20;
 			x ^= uint64_t(tv.tv_usec);
 		}
-		else {
-			x ^= uint64_t(seed) << 20;
-			x ^= uint64_t(seed);
-		}
```

Comment:
I agree theses two lines make no sense whatsoever (seed being bool true/false), this will make new icebram incompatible with the old one (different files generated for the same seed) ... no idea if that's important or not.

The rest LGTM.